http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -151,16 +151,16 @@ class ClusterMonitorActor(val cluster: Cluster,
     */
   private def handleReadyCluster(publicIp: IP, instances: Set[Instance]): Future[ClusterMonitorMessage] = {
     for {
-      clusterStatus <- getDbClusterStatus
+      cluster <- getDbCluster
```

Comment:
original code seems to just getting `Status`, any reason changing it to getting the whole `Cluster`?